enrollment-server/src/main/resources/application-dev.properties

```diff
@@ -0,0 +1,8 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/powerauth
+spring.datasource.username=
```

Comment:
Could we agree on some defaults on our dev machines? 

E.g. I am using `powerauth:Passw0rd` in all locally run databases.

Reply:
Imho if we add something like `-dev` props they should be able to be used without any change to run the app.

Reply:
I guess it is OK on this project, but on public projects, I am strongly against storing default passwords in `application.properties`, because it can inspire clients to use such passwords. I think a better approach may be sharing the Tomcat XML configurations in a private repo for all our projects.

Reply:
I recommend keeping the default username `powerauth` and empty password, as we do in any other projects

Reply:
In malwarelytics there is something like:
```
spring.datasource.username=${JDBC_DATABASE_USERNAME:#{null}}
spring.datasource.password=${JDBC_DATABASE_PASSWORD:#{null}}
```

Reply:
The idea behind is to have a custom variables (not related to spring) which holds the credentials and are referenced where needed.

Reply:
IMHO directive `#{null}` is overhead, we fail fast in case of invalid configuration.